rs/bitcoin/kyt/src/state.rs

```diff
@@ -86,10 +86,12 @@ impl TryFrom<Transaction> for TransactionKytData {
             .collect();
         let mut outputs = Vec::new();
         for output in tx.output.iter() {
-            outputs.push(Address::from_script(
-                &output.script_pubkey,
-                bitcoin::Network::Bitcoin,
-            )?)
+            // Some outputs do not have addresses. These outputs will never be
+            // inputs of other transactions, so it is okay to treat them as `None`.
+            outputs.push(
+                Address::from_script(&output.script_pubkey, bitcoin::Network::from(*btc_network))
+                    .ok(),
```

Comment:
nit: should we maybe log the errors?

Reply:
Thanks! Will add logging as part of https://dfinity.atlassian.net/browse/XC-205.